pallets/parachain-staking/src/delegation_requests.rs

```diff
@@ -349,7 +412,7 @@ impl<T: Config> Pallet<T> {
 					<T as Config>::WeightInfo::execute_delegator_revoke_delegation_worst();
 
 				// remove from pending requests
```

Comment:
```suggestion
				// remove from pending requests
				// `BoundedVec::remove` can panic, but we make sure it will not happen by checking above that `scheduled_requests` is not empty.
```